modules/microshift-storage-vol-snapshot-restore.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="restoring-vol-snapshot-microshift_{context}"]
+= Restoring a volume snapshot
+
+The following workflow demonstrates snapshot restoration. In this example, the verification steps are also given to ensure that data written to a source `PersistentVolumeClaim` (PVC) is preserved and restored on a new PVC.
```

Comment:
```suggestion
The following workflow demonstrates snapshot restoration. In this example, the verification steps are also given to ensure that data written to a source persistent volume claim (PVC) is preserved and restored on a new PVC.
```
See the repo [glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#persistent-volume-claim-pvc)